frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx

```diff
@@ -148,46 +105,69 @@ const SelectCapacityAndNodes: React.FC<SelectCapacityAndNodesProps> = ({
 type SelectCapacityAndNodesProps = {
   dispatch: WizardDispatch;
   capacity: WizardState['capacityAndNodes']['capacity'];
-  nodes: WizardState['capacityAndNodes']['nodes'];
+  nodes: WizardState['nodes'];
 };
 
 const SelectedCapacityAndNodes: React.FC<SelectedCapacityAndNodesProps> = ({
   capacity,
   storageClassName,
   enableArbiter,
+  arbiterLocation,
   dispatch,
   nodes,
 }) => {
   const { t } = useTranslation();
   const [pv, pvLoaded, pvLoadError] = useK8sWatchResource<K8sResourceKind[]>(pvResource);
   const memoizedPv = useDeepCompareMemoize(pv);
   const [allNodes, allNodeLoaded, allNodeLoadError] = useK8sWatchResource<NodeKind[]>(nodeResource);
-  const memoizedNodes = useDeepCompareMemoize(allNodes);
+  const memoizedAllNodes = useDeepCompareMemoize(allNodes);
```

Comment:
```suggestion
  const memoizedAllNodes = useDeepCompareMemoize(allNodes, true);
```
Object references are not going to be equal. Use string comparison please make changes at line 121 as well.